easybuild/easyconfigs/i/igraph/igraph-0.9.4-foss-2021a.eb

```diff
@@ -0,0 +1,51 @@
+# Author: Denis Krišťák (INUITS)
+# Modified: Jasper Grimm (UoY)
+
+easyblock = 'CMakeMake'
+
+name = 'igraph'
+version = '0.9.4'
+
+homepage = 'https://igraph.org'
+description = """igraph is a collection of network analysis tools with the emphasis on
+efficiency, portability and ease of use. igraph is open source and free. igraph can be
+programmed in R, Python and C/C++."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/igraph/igraph/releases/download/%(version)s']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['a3285cccf4f043c9ced2bc8d8d2609ff8398cb92ed49fdf86264ed91929137dd']
+
+builddependencies = [
+    ('Autotools', '20210128'),
+    ('pkg-config', '0.29.2'),
+    ('CMake', '3.20.1'),
+]
+
+dependencies = [
+    ('GLPK', '5.0'),
+    ('libxml2', '2.9.10'),
+    ('zlib', '1.2.11'),
+    ('SuiteSparse', '5.10.1', '-METIS-5.1.0'),
+    ('arpack-ng', '3.8.0'),
+]
+
+# Build static and shared libraries
+configopts = ["-DBUILD_SHARED_LIBS=OFF", "-DBUILD_SHARED_LIBS=ON"]
+
+# extra folder `igraph` is wrong, since all other software expects it to be in ./include folder directly
+postinstallcmds = [
+    'mv %(installdir)s/include/igraph/* %(installdir)s/include/ && rmdir %(installdir)s/include/igraph ',
+]
+
+_includes = ['include/igraph%s.h' % x for x in ['', '_blas', '_constants', '_lapack', '_types', '_version']]
+_libs = ['lib/libigraph.%s' % x for x in ['a', SHLIB_EXT]]
+
+sanity_check_paths = {
+    'files': _includes + _libs,
+    'dirs': [],
+}
```

Comment:
Is there a `sanity_check_command` we can run? Doesn't matter if not.

Reply:
I guess not...

Reply:
Not sure, I'm on leave and don't have my laptop with me so can't check

Reply:
Yes, not important - just nice to add more of these checks as we go.